### PR TITLE
Adding additional config/utility.

### PR DESCRIPTION
**Pull request by Ragundah**
Modified header to be cleaner.
Modified config layout.
Added following config options:
- Config_mailStationary
- Config_maxGMRank
- Config_preventReturn
- Config_charactersDbName

Renamed senderGUID to Config_senderGUID
Added function PreventReturn(playerGUID)
Tested and confirmed viability and functionality of additions and modifications.

### CODE REVIEW

**Review by 55Honey — CHANGES_REQUESTED** [review]
I like the additional config flag options, but if you want to change the header besides adding your name, you need to do that in your fork.

**Comment by Ragundah** [comment]
Corrected and added name and update date under original header.

File: LevelUpReward.lua [hunk 1]
```diff
@@ -37,17 +39,26 @@ Config_ItemAmount[35] = 10
 Config_ItemId[39] = 9312
 Config_ItemAmount[39] = 5
 
-local Config_mailText = 2
-local senderGUID = 1
+-- General Settings Config
+local Config_mailText = 1           -- Which text to send in the mail to the player.
```
Comment (55Honey) [inline LevelUpReward.lua]: I want the default config to stay, so this needs to be 2

File: LevelUpReward.lua [hunk 2]
```diff
@@ -37,17 +39,26 @@ Config_ItemAmount[35] = 10
...
+local Config_senderGUID = 1         -- GUID of the player found in CharactersSchema/Characters in Database.
+local Config_mailStationery = 61    -- Stationary used in the mail sent to the player. (41 Normal Mail, 61 GM/Blizzard Support, 62 Auction, 64 Valentines, 65 Christmas) Note: Use 62, 64, and 65 At your own risk.
```
Comment (55Honey) [inline LevelUpReward.lua]: same as above, must be 41 for the "Create a permanent copy of the letter" to work

**Review by 55Honey — CHANGES_REQUESTED** [review]

**Comment by Ragundah** [comment]
Dont merge yet. An issue cropped up relating to crashing.
=

**Comment by Ragundah** [comment]
Ready for any additional feedback/review as well as testing. Crash has been fixed. 62 64 and 65 stationary NOT linked to crashing it seems, but modifies the stationary in a way that could be undesirable. Leaving the text Note: Use 62, 64, and 65 At your own risk. for this reason.

File: LevelUpReward.lua [hunk 3]
```diff
@@ -162,4 +181,4 @@ local function GrantReward(event, player, oldLevel)
     end
 end
 
-RegisterPlayerEvent(PLAYER_EVENT_ON_LEVEL_CHANGE, GrantReward)
+RegisterPlayerEvent(PLAYER_EVENT_ON_LEVEL_CHANGE, GrantReward)
```
Comment (55Honey) [inline LevelUpReward.lua]: missing new line.

**Review by 55Honey — CHANGES_REQUESTED** [review]
Else looks good. I'll test it today.

**Pull request merged**